Orks.cat

```diff
@@ -8801,7 +8801,7 @@ When rolling to wound this unit, use the Big Mek’s Toughness while it is on th
             <characteristic name="S" typeId="59b1-319e-ec13-d466">2D6</characteristic>
             <characteristic name="AP" typeId="75aa-a838-b675-6484">-4</characteristic>
             <characteristic name="D" typeId="ae8a-3137-d65b-4ca7">4</characteristic>
-            <characteristic name="Abilities" typeId="837d-5e63-aeb7-1410">Before firing this weapon, roll to determine the Strength of the shot.  If the result is 12, do not make a wound roll.  Instead, if the attack hits, each cause 3 mortal wounds.  Then the bearer suffers a mortal wound.</characteristic>
+            <characteristic name="Abilities" typeId="837d-5e63-aeb7-1410">Before firing this weapon, roll to determine the Strength of the shot.  If the result is 12, do not make a wound roll.  Instead, if the attack hits, each causes 3 mortal wounds.  Then the bearer suffers a mortal wound.</characteristic>
```

Comment:
Wording in the Index is "Instead, if the attacks hit, each cause 3 mortal wounds".  "Cause" should still be "causes", but we should probably match GW's wording.